Dockerfile

```diff
@@ -52,9 +47,8 @@ RUN \
   phpenmod drupal-recommended && \
   phpdismod xdebug && \
   # Drush 8 is the current stable that supports Drupal version 6, 7 and 8.
-  curl -sS https://getcomposer.org/installer | sudo php -- --install-dir=/usr/local/bin --filename=composer && \
-  composer global require drush/drush:8.* && \
-  apt-get clean && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+  curl -sS https://getcomposer.org/installer | php -- --install-dir=/usr/local/bin --filename=composer && \
```

Comment:
How about [copying it from the Composer image](https://hub.docker.com/_/composer) instead as suggested in their documentation?

Reply:
Fine idea. See 88f61fe23bce49c2b8e8e30ed3983f10f93b77d9.